### ISSUE TITLE
4.4.2 b/c breaking ->getDoctrine failing... 

### ISSUE THREAD

**Issue by PhilETaylor** [symfony/symfony#35057]
Live that was fine with 4.4.1, composer update to 4.4.2 and deployed to production (too quick without testing) and production fell over in places. Doh.

Error is:

```
The DoctrineBundle is not registered in your application. Try running "composer require symfony/orm-pack".
```

I can replicate in dev mode too:

![Screenshot 2019-12-19 at 20 58 41](https://user-images.githubusercontent.com/400092/71209058-61424200-22a2-11ea-9b21-a4141e9fecf9.png)

bundles.php (truncated) does contain the DoctrineBundle

```
return [
    Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
    Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
    Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
    Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true, 'test' => true],
    Symfony\Bundle\MakerBundle\MakerBundle::class => ['dev' => true],
    Doctrine\Bundle\DoctrineCacheBundle\DoctrineCacheBundle::class => ['all' => true],
    Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle::class => ['all' => true],
    Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
```

**Comment by PhilETaylor** [comment]
Unable to reproduce with a new skeleton app :( 

Can reproduce in dev, debugging gives

```
d830b88c5319:/var/www/current# sf deb:container doctrine

Information for Service "doctrine"
==================================

 References all Doctrine connections and entity managers in a given Container.

 ---------------- -----------------------------------------
  Option           Value
 ---------------- -----------------------------------------
  Service ID       doctrine
  Class            Doctrine\Bundle\DoctrineBundle\Registry
  Tags             -
  Public           yes
  Synthetic        no
  Lazy             no
  Shared           yes
  Abstract         no
  Autowired        no
  Autoconfigured   no
 ---------------- -----------------------------------------
```

**Comment by alexsegura** [comment]
I just fust faced this too, tests suddenly broken when bumping Symfony to `4.4.2`. 
I fixed it by bumping DoctrineBundle from `1.12.2` to `1.12.5`.

**Comment by PhilETaylor** [comment]
hmm looks like Im using 2.0.2 aliased to 1.11.2 probably to overcome constraints in 

` "doctrine/doctrine-bundle": "2.0.2 as 1.11.2",`

bumping it to 

` "doctrine/doctrine-bundle": "2.0.6 as 1.11.2",`

resolved the issue for me - thanks for the tip... 

But still, maybe Symfony Devs might want to review this, as it might be a b/c break - or it might just be me being an idiot and pinning a set version of the bundle. 

I'll leave it open for now - anyone feel free to close it - panic over for me :)

**Comment by noniagriconomie** [comment]
@PhilETaylor you may edit your post above with the solution :)
got it also on a personal project :)

**Comment by xabbuh** [comment]
Could anyone of you still create a small example application that allows to reproduce the issue?

**Comment by xabbuh** [comment]
I am going to close here for now due to the lack of feedback. Please let us know when you have more information and we can consider to reopen.

**Comment by PhilETaylor** [comment]
Sorry for my delay. On holiday 4500miles from my desk. :-) back on 6th January.

**Comment by xabbuh** [comment]
@PhilETaylor Let's reopen then and reevaluate next year. Enjoy your holidays. :)

**Comment by PhilETaylor** [comment]
There is nothing to fix as far as I can see, as using the latest versions of everything, and using orm-pack instead of self-pinning a version of DoctrineBundle works perfectly.

The problem is only if people (like me) were hard coding versions of DoctrineBundle

**Comment by diegovidal27** [comment]
> I just faced this too, tests suddenly broken when bumping Symfony to `4.4.2`.
> I fixed it by bumping DoctrineBundle from `1.12.2` to `1.12.6`.

Thank you very much, it worked for me. I was just having trouble with the composer.lock, so I removed it and ran 'composer update' and voila!
